www/install/createTables.sql

```diff
@@ -1626,15 +1626,15 @@ CREATE TABLE `nagios_server` (
   `centreonbroker_cfg_path` varchar(255) DEFAULT NULL,
   `centreonbroker_module_path` varchar(255) DEFAULT NULL,
   `centreonconnector_path` varchar(255) DEFAULT NULL,
-  `ssh_port` int(11) DEFAULT NULL,
-  `ssh_private_key` varchar(255) DEFAULT NULL,
+  `gorgone_communication_type` enum('1', '2') NOT NULL DEFAULT '1',
+  `gorgone_port` int(11) DEFAULT NULL,
```

Comment:
just a comment -> why not 5556

Reply:
https://github.com/centreon/centreon/blob/master/www/install/createTables.sql#L1629